_includes/footer.html

```diff
@@ -1,32 +1,32 @@
 <footer class="site-footer">
   <div class="row fixed-width">
-    <div class="d-4-12 m-2-12">
+    <div class="column d-4-12 m-2-12">
       <h4>For Competitors</h4>
       <ul class="sitemap">
         <li><a href="/ide/">IDE</a></li>
         <li><a href="/docs/">Documentation</a></li>
         <li><a href="/forum/">Forums</a></li>
       </ul>
     </div>
-    <div class="d-4-12 m-2-12">
+    <div class="column d-4-12 m-2-12">
       <h4>Student Robotics</h4>
       <ul class="sitemap">
-        <li><a href="/about">About Us</a></li>
-        <li><a href="/compete">Compete</a></li>
-        <li><a href="/volunteer">Volunteer</a></li>
+        <li><a href="{{ '/about/' | prepend:site.baseurl }}">About Us</a></li>
+        <li><a href="{{ '/compete/' | prepend:site.baseurl }}">Compete</a></li>
+        <li><a href="{{ '/volunteer/' | prepend:site.baseurl }}">Volunteer</a></li>
       </ul>
     </div>
-    <div class="d-4-12 m-2-12">
+    <div class="column d-4-12 m-2-12">
       <h4>&nbsp;</h4>
       <ul class="sitemap">
-        <li><a href="/news">News</a></li>
-        <li><a href="/events">Events</a></li>
-        <li><a href="/contact">Contact us</a></li>
+        <li><a href="{{ '/news/' | prepend:site.baseurl }}">News</a></li>
+        <li><a href="{{ '/events/' | prepend:site.baseurl }}">Events</a></li>
+        <li><a href="{{ '/contact/' | prepend:site.baseurl }}">Contact us</a></li>
```

Comment:
Just noticed this having hit a similar issue on the competition-website pages: prepending this here results in these urls being "...org/docs/contact/" (etc.), which is clearly not desired (since they now 404).